src/extensions/mosip/MOSIPVerifiableCredential.yaml

```
@@ -1,43 +1,43 @@
 type: object
 description: MOSIP Verifiable Credential for a [person](https://github.com/opencrvs/mosip-mediator/blob/master/samples/decrypted-sample-received-credentials.json)
 properties:
-  issuedTo:
+  issued_to:
     description: ""
     type: string
-  protectedAttributes:
+  protected_attributes:
     type: array
     items:
       type: string
-  credentialSubject:
+  credential_subject:
     description: "MOSIP Verifiable Credential for Proof of identity"
     type: object
     properties:
       gender:
         $ref: LanguageValueList.yaml
       city:
         $ref: LanguageValueList.yaml
-      postalCode:
+      postal_code:
         type: string
         example: "14022"
-      fullName:
+      full_name:
         type: string
         example: "Thirteen Mosip"
-      dateOfBirth:
+      dateofbirth:
```

```suggestion
      date_of_birth:
```
or
```suggestion
      birthdate:
```
or is this MOSIP related so should be dateofbirth?